src/components/Card/Card.jsx

```diff
@@ -1,72 +1,58 @@
-import React from 'react'
+import React from 'react';
+import "./Card/Card.css";
+
 
 export const Card = () => {
```

Comment:
This is a bit confusing to read as there's an outer `Card` component that renders several `Card` components. Could the outer component be called `Cards` ?